client/my-sites/stats/site.jsx

```diff
@@ -615,16 +615,16 @@ class StatsSite extends Component {
 							)
 						}
 
-						{ supportsDevicesStats && ! isOldJetpack && (
+						{ supportsDevicesStats && (
 							<StatsModuleDevices
 								siteId={ siteId }
 								period={ this.props.period }
 								query={ query }
 								className={ halfWidthModuleClasses }
 							/>
 						) }
-
-						{ ! supportsDevicesStats && isOldJetpack && (
```

Comment:
In the old code, `supportsDevicesStats` means `isJetpackNotAtomic`, and `isOldJetpack` means `statsAdminVersion`<=`0.19.0-alpha`, which makes the update plugin cta to show when site is (Atomic or Simple) and stats admin package is outdated.